Source/OxyPlot/Series/DataPointHelper.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections.Generic;
+
+namespace OxyPlot.Series
+{
+	/// <summary>
+	/// Data point helper functions.
+	/// </summary>
+	public class DataPointHelper
```

Comment:
I think this class/method belongs under the Utilities folder. Or could it be a protected member under the DataPointSeries?